crates/sparrow-main/src/prepare.rs

```diff
@@ -179,7 +179,7 @@ impl PrepareCommand {
                     PulsarSubscription {
                         config: Some(pulsar_config),
                         subscription_id: pulsar_subscription,
-                        last_publish_time: None,
+                        last_publish_time: 0,
```

Comment:
FWIW, I have seen many problems with systems that use `0` to represent "the time before everything else"... we may want to handle "what if something has a time of 0"... but it's probably OK?

Reply:
this will eventually come from wren, instead of the current hard coded value.